public/admin/config.yml

```diff
@@ -99,6 +85,20 @@ collections:
           pattern: ["^.{50,160}$", "Must be between 50 and 160 characters"],
           i18n: true,
         }
+      - {
+          label: "Category",
```

Comment:
Did you also assign these to the existing guides in markdown?

Reply:
yes, for the stories that are in the repo